Mathlib/Tactic/Measurability.lean

```diff
@@ -29,7 +29,7 @@ macro "measurability" : attr =>
 The tactic `measurability` solves goals of the form `Measurable f`, `AEMeasurable f`,
 `StronglyMeasurable f`, `AEStronglyMeasurable f μ`, or `MeasurableSet s` by applying lemmas tagged
 with the `measurability` user attribute. -/
-macro "measurability" (config)? : tactic =>
+macro "measurability" _cfg:optConfig : tactic =>
```

Comment:
Why the `_cfg` here?

Reply:
These tactics apparently don't use their configurations. The reason the syntax worked before is that `macro` will turn the parser `(config)?` into something like `_:(config)?`

Reply:
I think I'll delete the configuration actually. I was going for doing one-to-one changes with this PR, but having an unused configuration is bound to be confusing to someone.